controller-manifests/cluster-logging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,361 @@
+          - replicasets
+          - statefulsets
+          verbs:
+          - "*"
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          - routes/custom-host
+          verbs:
+          - "*"
+        - apiGroups:
+          - batch
+          resources:
+          - cronjobs
+          verbs:
+          - "*"
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - roles
+          - rolebindings
+          verbs:
+          - "*"
+        - apiGroups:
+          - security.openshift.io
+          resources:
+          - securitycontextconstraints
+          resourceNames:
+          - privileged
+          verbs:
+          - use
+      clusterPermissions:
+      - serviceAccountName: cluster-logging-operator
+        rules:
+        - apiGroups:
+          - scheduling.k8s.io
+          resources:
+          - priorityclasses
+          verbs:
+          - "*"
+        - apiGroups:
+          - oauth.openshift.io
+          resources:
+          - oauthclients
+          verbs:
+          - "*"
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterroles
+          - clusterrolebindings
+          verbs:
+          - "*"
+      deployments:
+      - name: cluster-logging-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: cluster-logging-operator
+          template:
+            metadata:
+              labels:
+                name: cluster-logging-operator
+            spec:
+              serviceAccountName: cluster-logging-operator
+              containers:
+              - name: cluster-logging-operator
+                image: quay.io/openshift/origin-cluster-logging-operator:latest
+                imagePullPolicy: IfNotPresent
+                command:
+                - cluster-logging-operator
+                env:
+                  - name: WATCH_NAMESPACE
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.annotations['olm.targetNamespaces']
+                  - name: OPERATOR_NAME
+                    value: "cluster-logging-operator"
+                  - name: ELASTICSEARCH_IMAGE
+                    value: "quay.io/openshift/origin-logging-elasticsearch5:latest"
+                  - name: FLUENTD_IMAGE
+                    value: "quay.io/openshift/origin-logging-fluentd:latest"
+                  - name: KIBANA_IMAGE
+                    value: "quay.io/openshift/origin-logging-kibana5:latest"
+                  - name: CURATOR_IMAGE
+                    value: "quay.io/openshift/origin-logging-curator5:latest"
+                  - name: OAUTH_PROXY_IMAGE
+                    value: "quay.io/openshift/origin-oauth-proxy:latest"
+                  - name: RSYSLOG_IMAGE
+                    value: "docker.io/viaq/rsyslog:latest"
+  version: 0.0.1
```

Comment:
i think this should be `v4.0.0` right now. 

/cc @jwforres

Reply:
@bparees updated to 4.1.0